web/templates/topic_sidebar_actions.hbs

```diff
@@ -11,13 +11,22 @@
     {{#if development}}
         <li class="hidden-for-spectators">
             <div class="tabs-container">
-                <div class="tab-option tippy-zulip-tooltip {{#if (eq visibility_policy all_visibility_policies.MUTED)}}selected-tab{{/if}}" data-visibility-policy="{{all_visibility_policies.MUTED}}" data-tippy-content="{{t 'Mute topic' }}" aria-label="{{t 'Mute topic' }}">
+                <div class="tab-option tippy-zulip-tooltip {{#if (eq visibility_policy all_visibility_policies.MUTED)}}selected-tab{{/if}}" data-visibility-policy="{{all_visibility_policies.MUTED}}" data-tippy-content="{{t 'Mute' }}" aria-label="{{t 'Mute' }}">
                     <i class="zulip-icon zulip-icon-mute-new"></i>
```

Comment:
I have changed the tippy-content from "Mute topic" to "Mute" because "Regular topic" doesn't look good ? (if we plan to keep things consistent).

I think, "Mute", "Unmute", "Follow", "Regular" sounds good and its obvious that we are performing action on the topic itself (as it is a topic popover)

Reply:
That makes sense to me

Reply:
Maybe we should change the labels in a separate commit though

Reply:
Done